### ISSUE TITLE
Where is the constrcit of Flops?

### ISSUE THREAD

**Issue by qtw1998** [toandaominh1997/EfficientDet.Pytorch#50]
EfficientNet can be constricted by FLOPS you know

**Comment by qtw1998** [comment]
![image](https://user-images.githubusercontent.com/34357479/71573939-3ae19980-2b21-11ea-8374-0092aef54e4d.png)
Maybe need pretrained weights?

**Comment by toandaominh1997** [comment]
@qtw1998 No, you need create folder 'weights' or add `os.makedirs(directory)' to create folder contains 'weights'. You do not need to pretrained the model first. Sorry, I have not added this in the code yet
How do you fix out memory in centos?. Please let me know??

**Comment by qtw1998** [comment]
> @qtw1998 No, you need create folder 'weights' or add `os.makedirs(directory)' to create folder contains 'weights'. You do not need to pretrained the model first. Sorry, I have not added this in the code yet
> How do you fix out memory in centos?. Please let me know??

![image](https://user-images.githubusercontent.com/34357479/71575805-446eff80-2b29-11ea-8916-995d6f07d706.png)
I'm using 8 x 2080ti to  train the model now. Maybe there are some bugs in the code, like some memory leakings?

**Comment by toandaominh1997** [comment]
@qtw1998 I see it only takes ~3000 MB per GPU, you try increasing the batch_size?

**Comment by toandaominh1997** [comment]
@qtw1998 Can you train COCO2017 and let me know mAP. I haven't had time to train this dataset now

**Comment by qtw1998** [comment]
> @qtw1998 Can you train COCO2017 and let me know mAP. I haven't had time to train this dataset now

Would you please upload the bash scripts(it's about the COCO2014)？hhh THX！！

**Comment by qtw1998** [comment]
> @qtw1998 I see it only takes ~3000 MB per GPU, you try increasing the batch_size?

NO, I don't know why. If I use one less, will have the same problem I have posted yesterdayRuntimeError: CUDA error: out of memory

**Comment by toandaominh1997** [comment]
I updated sell script for download coco2017 at (https://github.com/toandaominh1997/EfficientDet.Pytorch/blob/master/datasets/scripts/COCO2017.sh). Please checkout last commit and follow readme for training coco2017. Thanks

**Comment by toandaominh1997** [comment]
Can you increase the batch size(+4 or +8 ...) to test whether the model for
the percentage of gpu?

**Comment by qtw1998** [comment]
> I updated sell script for download coco2017 at (https://github.com/toandaominh1997/EfficientDet.Pytorch/blob/master/datasets/scripts/COCO2017.sh). Please checkout last commit and follow readme for training coco2017. Thanks @qtw1998

Thx bro, and do you use any other instant social medium, maybe we coulda had a better conservation if you'd like, you can send it to my email:qtw1998sjtuer@gmail.com. Then I can add you on Facebook, IG or even Wechat, etc... and we can have a comprehensive talk or maybe I need your halp! thx bro
